wp-seo-main.php

```diff
@@ -414,6 +414,10 @@ function wpseo_admin_init() {
 $register_capabilities = new WPSEO_Register_Roles();
 $register_capabilities->register_hooks();
 
+// Compatibility with Members.
+$capability_manager_integration = new WPSEO_Capability_Manager_Integration();
```

Comment:
Could you move this into the `WPSEO_Admin` class on a `plugins_loaded` hook?
This does not need to be loaded at this specific moment in the plugin.

The roles and capabilities can be registered quite early because it's just an internal registration, so they can be used whenever needed.

Reply:
@moorscode No problem. What do you think about making the class methods static?
There is no need for instances of this class.

Plus I'm not so fond of passing a new instance in a hook like this:
```php
add_action( 'plugins_loaded', array( new WPSEO_Capability_Manager_Integration, 'register_hooks' ) );
```

I think this would be better since it allows you deregister if needed:
```php
add_action( 'plugins_loaded', array( 'WPSEO_Capability_Manager_Integration', 'register_hooks' ) );
```

Reply:
Oh even better. In `WPSEO_Admin::__construct()` we can just call `WPSEO_Capability_Manager_Integration::register_hooks()` without an action since this class is already in the `plugins_loaded` action.

Reply:
I have a commit ready to push, let me know!